runtime/near-vm-logic/src/mocks/mock_memory.rs

```diff
@@ -2,28 +2,46 @@ use crate::{MemSlice, MemoryLike};
 
 use std::borrow::Cow;
 
-#[derive(Default)]
-pub struct MockedMemory {}
+pub struct MockedMemory(Box<[u8]>);
+
+impl Default for MockedMemory {
+    fn default() -> Self {
+        Self(vec![0; 64 * 1024].into())
```

Comment:
The reason for 64K is that tests for other memory back ends use that size (a single WASM page) and it’s what test_memory_like expects.  It could be done more flexible but I’d argue the current approach is sufficient for the test needs.

Reply:
I honestly don’t see why changing it to vector would be better.  And the refactor will be risky in the sense that test_memory_like test will break so it cannot be done just like that.